File: app/src/main/java/com/scatl/uestcbbs/module/credit/view/CreditHistoryView.java
package com.scatl.uestcbbs.module.credit.view;

import com.scatl.uestcbbs.entity.MineCreditBean;

import java.util.List;

/**
 * author: sca_tl
 * date: 2020/12/12 19:03
 * description:
 */
public interface CreditHistoryView {
    void onGetMineCreditHistorySuccess(List<MineCreditBean.CreditHistoryBean> creditHistoryBeans, boolean hasNext);
    void onGetMineCreditHistoryError(String msg);
}

File: app/src/main/java/com/scatl/uestcbbs/module/credit/view/CreditHistoryView.kt
package com.scatl.uestcbbs.module.credit.view

import com.scatl.uestcbbs.base.BaseView
import com.scatl.uestcbbs.entity.MineCreditBean.CreditHistoryBean

interface CreditHistoryView: BaseView {
    fun onGetMineCreditHistorySuccess(creditHistoryBeans: List<CreditHistoryBean>, hasNext: Boolean)
    fun onGetMineCreditHistoryError(msg: String?)
}